http_handlers.go

```diff
@@ -18,12 +18,13 @@ func healthz(c *gin.Context) {
 }
 
 func webhook(c *gin.Context) {
+	ctx := c.Request.Context()
```

Comment:
docs recommend using this as the context https://docs.bugsnag.com/platforms/go/gin/#using-context-context-to-manage-http-session-data